test/setter/grandchild.js

```diff
@@ -1,5 +1,5 @@
 export let c = 0;
 export function increment() {
   ++c;
-  module.export();
+  module.runSetters();
```

Comment:
This `module.export()` call was a very old way of triggering setters, even less common than `module.export("default", ...)`. Glad to see it gone!